examples/build_scripts/build_gaea_c5_gcc.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Example bash script to install Pace to run bare-metal on Gaea's c4 cluster
+
+set -e -x
+
+# module load necessary system software
+module rm PrgEnv-intel
+module load PrgEnv-gnu
+module rm gcc
+module load gcc/12.2.0
+module load boost/1.79.0
+module load python/3.9
```

Comment:
Should we update the python version here too?

Reply:
yes, updated to the python 3.11.7